easybuild/easyconfigs/c/COSMO_pompa/COSMO_pompa-31d7227-CrayCCE-18.08-cordex-double.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'MakeCp'
+
+name = 'COSMO_pompa'
+version = '31d7227'
+implementation = 'cordex'
+precision = 'double'
+versionsuffix = '-%s-%s' % (implementation, precision)
+
+homepage = 'https://github.com/C2SM-RCM/cosmo-pompa/tree/master/dycore (-b crclim)'
+description = """
+cosmo-pompa
+"""
+
+toolchain = {'name': 'CrayCCE', 'version': '18.08'}
+toolchainopts = {'verbose': False}
+# repo is private (+ no official release)
+sources = [
+    {'filename': '/apps/common/UES/easybuild/sources/c/COSMO_pompa_crclim/'    \
+        'cosmo_pompa_crclim_31d7227a03e90386fa9f7884e0c7bed4e2ae2a81.tar.gz'}
+]
+
+builddependencies = [
+    ('CMake', '3.12.0', '', True), 
+    ('libgrib1_crclim', 'a1e4271'),
+    ('Dycore/551f6b4-CrayGNU-18.08-%s-%s' 
```

Comment:
What's the difference of this Dycore to the `COSMO_pompa_crclim_dycore` one used above ? I see they both have the same commit hash.

Reply:
Dycore and COSMO_pompa_crclim_dycore are basically the same. I changed it, so that there was a more unique naming scheme. (also, because for me crclim and cordex are versions of COSMO (based on settings in STELLA) and therefore (IMO) should not be part of the name) 
This is more intended as a suggestion.